ui/store/background-connection.ts

```diff
@@ -39,8 +39,12 @@ type CallbackMethod<R = unknown> = (error?: unknown, result?: R) => void;
 
 /**
  * [Deprecated] Callback-style call to background method
- * invokes promisifiedBackground method directly.
+ * In MV2: invokes promisifiedBackground method directly.
+ * In MV3: action is added to retry queue, along with resolve handler to be executed on completion,
+ * the queue is then immediately processed if background connection is available.
+ * On completion (successful or error) the action is removed from the retry queue.
```

Comment:
Hmm, I don't think any of this is correct. In both MV2 and MV3 builds, this will submit the request to the background (though _not_ via `promisifiedBackground`, which also is not a method).

Reply:
they were migrated from old doc. thanks for pointing out, they are not accurate any more. removed now 🙏🏻